packages/bridge/jest.config.js

```diff
@@ -1,3 +1 @@
-module.exports = {
-  preset: 'ts-jest',
-};
+module.exports = require('../../jest.config');
```

Comment:
This config is not good for `@aloxide/bridge` model since it need to check `require.resolve('@aloxide/bridge')`.

Please revert this file.

Reply:
reverted